src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl

```diff
@@ -42,7 +42,7 @@ mutable struct HydrostaticFreeSurfaceModel{TS, E, A<:AbstractArchitecture, S,
           velocities :: U        # Container for velocity fields `u`, `v`, and `w`
              tracers :: C        # Container for tracer fields
             pressure :: Φ        # Container for hydrostatic pressure
-       diffusivities :: K        # Container for turbulent diffusivities
+  diffusivity_fields :: K        # Container for turbulent diffusivities
          timestepper :: TS       # Object containing timestepper fields and parameters
     auxiliary_fields :: AF       # User-specified auxiliary fields for forcing functions and boundary conditions
```

Comment:
The formatting got mangled (the longest name, which is now `diffusivity_fields`, should be indented by 4). These 20 lines or so need to be indented by 2 more spaces (please visually inspect to ensure it is readable).

Reply:
I'll do it, but just so I know: why is 4 the minimum indentation? Does it have to do with Documenter.jl when rendering?

Reply:
No, it's just our convention to use 4 space for indents. It has no effect on execution.